px4ctrl:
  ros__parameters:

    feedforward_thrust: true
    use_bodyrate: false
    publish_enabled: true

    ctrl_rate: 100.0
    mass: 1.0
    gra: 9.81

    kp_xy: 6.0
    kp_z: 6.0
    kv_xy: 4.0
    kv_z: 4.0
    kvi_xy: 0.1
    kvi_z: 0.2
    ka: 1.0
    kT: 1.0

    katt_xy: 6.0
    katt_z: 4.0

    max_tilt_deg: 35.0
    hover_thrust: 0.5
    thrust_min: 0.05
    thrust_max: 0.95
